install/error-handling.sh

```diff
@@ -1,10 +1,51 @@
 echo "${_group}Setting up error handling ..."
 
+export SENTRY_DSN='https://19555c489ded4769978daae92f2346ca@self-hosted.getsentry.net/3'
+export SENTRY_ORG=self-hosted
+export SENTRY_PROJECT=installer
+
+function send_event {
+  if [[ $DOCKER_PLATFORM == "linux/amd64" ]]; then
+    local sentry_cli="docker run --rm -v \$(pwd):/work -e SENTRY_DSN=$SENTRY_DSN getsentry/sentry-cli"
+  else
+    if ! command -v sentry-cli &> /dev/null; then
+        echo "sentry-cli could not be found, please install it"
+        exit 1
+    fi
+    local sentry_cli=sentry-cli
+  fi
+  command pushd .. > /dev/null
+  $sentry_cli send-event --no-environ -f "$1" -m "$2" --logfile $log_file
+  command popd > /dev/null
+}
+
+if [[ -f ../.reporterrors ]]; then
```

Comment:
This is insufficient, because we aren't guaranteed to be in the `install` directory.

https://github.com/getsentry/self-hosted/blob/013519a46c696438016a1577615d47f1ec677212/install/_lib.sh#L11-L16

I made a PR at #1686 to propose a significant copy update, and it includes a fix using `$basedir`. What do you think?